Brighter/paramore.brighter.serviceactivator/MessagePump.cs

```diff
@@ -303,12 +303,13 @@ private void RequeueMessage(Message message)
 
                     if (Logger != null) 
                         Logger.ErrorFormat(
-                            "MessagePump: Have tried {2} times to handle this message {0}{4} from {3} on thread # {1}, dropping message", 
+                            "MessagePump: Have tried {2} times to handle this message {0}{4} from {3} on thread # {1}, dropping message. Message Body: {5}", 
```

Comment:
Probably better to put in a Environment.NewLine so the body is on a new line. Will be easier to read then